bind_exporter.go

```diff
@@ -443,7 +446,7 @@ func (s *statisticGroups) String() string {
 	for _, g := range *s {
 		groups = append(groups, string(g))
 	}
-	return fmt.Sprintf("%q", strings.Join(groups, ","))
+	return strings.Join(groups, ",")
```

Comment:
Hmm, the `fmt.Sprintf()` bit is for string quoting safety. I wonder why it was like that.

Reply:
If we leave it as it was, go-kit logs it double-quoted, e.g. `collectors="\"foo,bar,bz\""`.

Reply:
Ahh, ok, that makes sense.